script/Ciel/Day 13/13_01_02_CIEL13_7B.txt

```diff
@@ -117,12 +123,12 @@ Anyone would've said the same, and they'd have been right to say it.
 [sha:5662aaf0369170a2860f52b076d7d01e2097b58f]{
 -- Page 238, Offset 35949.
 -- わたしなんかが、生きていていい理由はなかった。
-Someone like me had no reason to even live on.
+There was no reason someone like me deserved to go on living.
 }
 [sha:ff46b11c856d4c3766280399b02a7d51f4cb7d4b]{
 -- Page 239, Offset 35950.
 -- せめて一日で終わってほしかった。
-At the very least, I hoped that it'd all be over in a single day.
+I had hoped that it would at least be over in a day.
 }
 [sha:1a26ebb5d292420cb6df12a6439488a1df0e9de2]{
 -- Page 239, Offset 35951.
```

Comment:
That isn't really the correct collocation - day can dawn or break, but night doesn't. You could rewrite both sentences if you really wanted to - 
> I had hoped that it would at least be over by tomorrow―――
> But the next day never dawned.

Reply:
Yeah, on further thought, nobody on the internet seems to agree what "the night of dawn" means, despite it being a relatively attested phrase. What an incredibly weird dilemma!

Here's an option that is a bit more transformative

>I had hoped that it would at least be over in a night.

>Yet daybreak never came.